internal/cmds/cmds.go

```diff
@@ -295,6 +315,28 @@ func enable(ctx *log.Context, h types.HandlerEnvironment, report *types.RunComma
 	return stdoutTail, stderrTail, runErr, exitCode
 }
 
+func createExtensionEventManager(ctx *log.Context, hEnv types.HandlerEnvironment) *extensionevents.ExtensionEventManager {
+	el := logging.New(nil)
+	platformHandlerEnv := convertToPlatformHandlerEnv((hEnv))
+	extensionEvents := extensionevents.New(el, platformHandlerEnv)
+	return extensionEvents
+}
+
+func convertToPlatformHandlerEnv(myEnv types.HandlerEnvironment) *handlerenv.HandlerEnvironment {
```

Comment:
Please consider adding a comment what this function does, possibly with an example.